enkit/outputs/commands.go

```diff
@@ -55,26 +60,83 @@ func NewRoot(base *client.BaseFlags) (*Root, error) {
 type Mount struct {
 	*cobra.Command
 	root *Root
+
+	BuildBuddyApiKey  string
+	BuildBuddyUrl     string
+	ClusterName       string
+	DryRun            bool
+	InvocationID      string
+	SymlinkFormat     string
+	UseTestResults    bool
+	UseBuildArtifacts bool
```

Comment:
Can we drop these? (hardcode SymlinkFormat, drop UseTestResults/UseBuildArtifacts) - these don't have usecases as per our doc, and I'd rather not expand the scope until we have a reason)

Reply:
Dropped. keeping dry run for debugging however.